### DISCUSSION TITLE
我的绳网等级终于上50了！！！！！！！！！！！！！！！！

### DISCUSSION THREAD

**Discussion by shenguojia123** [share121/inter-knot#2217] [其他]
如题！发上来记录一下！
![c19d47d2352491c61cc4357a3b940c4](https://github.com/user-attachments/assets/24871cd6-95b8-4a1b-baf7-2dee02284cee)

**Comment by poolofwater** [comment]
恭喜！<img src="https://img-static.mihoyo.com/communityweb/upload/951272c9ce086ecb1baefd179f18973c.png" alt="庆贺吧" width="75" >

**Comment by Sheng-gy** [comment]
50级了啊所料不错你应该没钱了<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Reply by shenguojia123** [reply]
只剩两万多丁尼了
![5ae950d0-4ee8-4c7f-943d-e72271940fca](https://github.com/user-attachments/assets/99b8eae7-be6b-48ca-be73-eab798c45e7e)

**Comment by shenguojia123** [comment]
那个那个，你们也可以加一下我啊<img src="https://bbs-static.miyoushe.com/static/2024/07/31/fdc5e0e30e577fb208f29effa93510ed_8442612635207222552.png" alt="云璃 期待" width="75" >

**Comment by tingsha1935** [comment]
太能肝了

**Comment by ding341** [comment]
神郭嘉还挺勤奋

**Comment by Phospherus9484** [comment]
加油哦小可爱
